tests/test_coco_format.py

```diff
@@ -836,6 +848,205 @@ def test_can_pickle(self):
                 compare_datasets_strict(self, source, parsed)
 
 
+class CocoExtractorTests(TestCase):
+    ANNOTATION_JSON_TEMPLATE = {
```

Comment:
IMO, it would be useful to have a test that loads the unmodified template, to ensure that any errors in the other tests are due to the modifications made in those tests and not due to the template itself.

Reply:
The fact that the common piece was possible to extract is occasional, it is not something designed or expected. There are lots of tests that cover successful loading.

Reply:
> The fact that the common piece was possible to extract is occasional, it is not something designed or expected.

How is that not expected? The point of the new tests is to make sure that specific errors in the dataset cause the extraction to fail. To be certain that it's _those_ errors that cause the failure, we need to ensure that the original template contains no errors.

Reply:
It is checked by the error details checks.